docs/persistence-and-the-mysql-backend.md

```diff
@@ -1,5 +1,5 @@
 ---
-category: Develop
+category: API Reference
 ---
 # Database schema
```

Comment:
Moved database schema to reference category as it is a reference and not a guide